### ISSUE TITLE
Re-review large scale job handling page

### ISSUE THREAD

**Issue by Pratichhya** [eu-cdse/documentation#591]
Revisit certain sections on this page: https://documentation.dataspace.copernicus.eu/APIs/openEO/large_scale_processing.html for further updates

**Comment by HansVRP** [comment]
To read through documentation and make recommendations

**Comment by Pratichhya** [comment]
Possible changes to this page:

- Rearrange the description of Large-scale processing in openEO in the introductory paragraph
- Rearrange the showcase example and related topic in a separate section. 
- Is preparing the tiling grid and job attributes relevant for each large-scale processing?
- Within the Tuning job options section, how can a general user tune the job options? 
- Also, the title [start map production](https://documentation.dataspace.copernicus.eu/APIs/openEO/large_scale_processing.html#starting-map-production)still shows that this is about this specific showcase rather than the [MultiBackendJobManager](https://open-eo.github.io/openeo-python-client/cookbook/job_manager.html#openeo.extra.job_management.MultiBackendJobManager). So maybe rephrase the title. Then, focusing back on the usecase after defining it in general might be easier
